importers/ableroad.com/easy_importer.py

```diff
@@ -0,0 +1,226 @@
+    if not m:
+        return ''
+    return m.group('name')
+
+def getcategory(s):
+    if not len(s):
+        return ''
+    return re.split(r':\s*',s[0])[1]
+
+def getdistance(s):
+    if not len(s):
+        return ''
+    reg = r'(?P<dis>\d+\.?\d*)'
+    m = re.search(reg,s[0])
+    if not m:
+        return ''
+    #conver mile to km
+    dis = m.group('dis')
+    dis = float(dis) * 1.60934
+    dis = str(round(dis, 2))
+    return dis
+
+def getneighborhood(s):
+    if not len(s):
+        return ''
+    return re.split(r':\s*',s[0])[1]
+
+def getableroad(s):
+    if not len(s):
+        return ''
+    reg = r'(?P<ablerating>\d+\.?\d*)'
+    m = re.search(reg,s[0])
+    if not m:
+        return ''
+    return m.group('ablerating')
+
+def getyelp(s):
+    if not len(s):
+        return ''
+    reg = r'(?P<yelp>\d+\.?\d*)'
+    m = re.search(reg,s[0])
+    if not m:
+        return ''
+    return m.group('yelp')
+
+def getstreet(s):
+    if not len(s):
+        return ''
+    return s[0]
+
+def getcity(s):
+    if len(s) < 1:
+        return ''
+    s = ' '.join(s)
+    m = re.search(r'(?P<city>[a-zA-Z]+),\s*', s)
+    if not m:
+        return ''
+    return m.group('city')
+
+def getstate(s):
+    if not len(s):
+        return ''
+    s = ' '.join(s)
+    m = re.search(r'(?P<city>[a-zA-Z]+),\s*(?P<state>[a-zA-Z]+)', s)
+    if not m:
+        return ''
+    return m.group('state')
+
+def getpostcode(s):
+    if not len(s):
+        return ''
+    s = ' '.join(s)
+    m = re.search(r'(?P<city>[a-zA-Z]+),\s*(?P<state>[a-zA-Z]+)\s+(?P<post>(\d{5,5})|(\w{3}\s\w{3}))', s)
+    if not m:
+        return ''
+    return m.group('post')
+
+def getphone(s):
+    if len(s) < 1:
+        return ''
+    if not re.search(r'\d{2,}-\d+',s[-1]):
+        return ''
+    return s[-1]
+
+def extract_info(dombus):
+    row = []
+    name = dombus.xpath('.//a[@class="titlelink"]/text()')
+    name = getname(name)
+    row.append(name)
+
+    category = dombus.xpath('.//div[@class="category"]/text()')
+    category = getcategory(category)
+    row.append(category)
+
+    distance = dombus.xpath('.//div[@class="itemdistance"]/text()')
+    distance = getdistance(distance)
+    row.append(distance)
+
+    neighborhood = dombus.xpath('.//div[@class="neighborhood"]/text()')
+    neighborhood = getneighborhood(neighborhood)
+    row.append(neighborhood )
+
+    ableroadating = dombus.xpath('.//div[@class="searchableRating"]//div[@class="visually-hidden startableft"]/text()')
+    ableroadating = getableroad(ableroadating)
+    row.append(ableroadating)
+
+    yelprating = dombus.xpath('.//img[@class="yelprating"]/@alt')
+    yelprating = getyelp(yelprating)
+    row.append(yelprating)
+
+    address = dombus.xpath('.//address/text()')
+
+    street = getstreet(address)
+    row.append(street)
+
+    city = getcity(address)
+    row.append(city)
+
+    state = getstate(address)
+    row.append(state)
+
+    postcode = getpostcode(address)
```

Comment:
Obviously the function name works but I want to follow PEP-8 coding style conventions for our Python code.  underscores should separate words in your Python function names.  For example get_postal_code instead of getpostcode.

Replace all the function names with that format.

More details are at: https://www.python.org/dev/peps/pep-0008/#function-names